delivery_transsmart/__scripts__/res_partner.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# TODO: go through all the res.partners. if they do not have a value in those
```

Comment:
Instead of a script you might also make this part of a migration script, or have it run in an init hook. So will be fully automated.